dist/polyfill.object-fit.js

```diff
@@ -36,9 +36,22 @@
 		return items;
 	};
 
+	// get host of stylesheet
+	var getHost = function(href) {
```

Comment:
Please follow the existing coding conventions: `function (href)`.
I’d also prefer to have a better function name like `getCSSHost` as it’s currently unknown what host it fetches.